docker-compose.yaml

```diff
@@ -38,5 +38,7 @@ services:
       - video-gateway
     env_file: .env
   video-gateway:
+    environment:
+      - PARSER_TYPE=ffmpeg
```

Comment:
Does video gateway use parser? Even if yes - just put it to the .env file

Reply:
glad that you have good eyes.. :D